src/pages/GosuDetail/GosuDetail.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+import GosuMainSection from './GosuMainSection/GosuMainSection';
+import GosuAsideBar from './GosuAsideBar/GosuAsideBar';
+
+function GosuDetail() {
+  const [gosuDetails, setGosuDetails] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/gosuDetail.json')
+      // fetch('http://192.168.0.103:8000/masters/profile/1')
+      .then(res => res.json())
+      .then(data => setGosuDetails(data[0]));
+    // .then(data => setGosuDetails(data.data[0]));
+  }, []);
+
+  return (
+    <GosuDetailContainer>
+      {gosuDetails.length !== 0 && (
+        <GosuMainSection gosuDetails={gosuDetails} />
+      )}
+      {gosuDetails.length !== 0 && <GosuAsideBar gosuDetails={gosuDetails} />}
```

Comment:
```suggestion
      {gosuDetails.length && (
        <>
          <GosuMainSection gosuDetails={gosuDetails} />
          <GosuAsideBar gosuDetails={gosuDetails} />
        </>
      )}
```
작성해주신 방법은 같은 조건을 두번 체크하고 있습니다. gosuDetails의 길이를 한번만 체크하는 방식으로 바꿔주세요.